lib/xctest.js

```diff
@@ -0,0 +1,241 @@
+import { startHouseArrestService, startTestmanagerdService, startInstallationProxyService, startInstrumentService } from './services';
+import { INSTRUMENT_CHANNEL } from './instrument';
+import { TESTMANAGERD_CHANNEL } from './testmanagerd';
+import { DTXMessageAuxBuffer } from './instrument/headers';
+import { NSUUID, XCTestConfiguration } from './instrument/transformer/nskeyed';
+import { uuidV4 } from '@appium/support/build/lib/util';
```

Comment:
Please avoid such imports. Import it like in https://github.com/appium/appium-xcuitest-driver/blob/70b85b24135a7aa585818f5bb4b795ea9322d93e/lib/commands/pcap.js#L110